### PR TITLE
Task #2546 - Add plyr player to movie player

### PR DESCRIPTION
**Pull request by ndviet2307**

### CODE REVIEW

**Comment by ndviet2307** [comment]
screenshot from 2018-10-13 17-55-29

**Comment by ndviet2307** [comment]
ready

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 //= require jquery3
 //= require popper
 //= require bootstrap-sprockets
+//= require plyr/dist/plyr.min
```
Comment (thanhhungchu95) [inline app/assets/javascripts/application.js]: cái này là dùng nodejs cài đấy à (yaoming)

**Comment by thanhhungchu95** [comment]
commented

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (ndviet2307) [reply app/assets/javascripts/application.js]: yep :+1:

**Comment by ndviet2307** [comment]
screenshot from 2018-10-15 08-10-55

**Comment by lyvanminh** [comment]
OK

**Comment by ndviet2307** [comment]
ready

**Pull request merged**